docs/api.asciidoc

```diff
@@ -319,8 +318,8 @@ does not contain a span.
 [[context-api]]
 === Context
 
-When reporting transactions and errors you can provide context to describe
-those events. Built-in instrumentation will typically provide some context,
+Use context to describe transaction reporting and errors.
+ Built-in instrumentation will typically provide some context,
 e.g. the URL and remote address for an HTTP request. You can also provide
 custom context and tags.
 
```

Comment:
Line 343, Google says Boolean should be capitalized and we capitalized it in some of the earlier articles I read.